### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #23082: Contributor License Agreement link broken
The [link to the CLA](https://grafana.com/docs/contribute/cla/) from the [contributing guide]( https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md) is being redirected to https://grafana.com/docs/grafana/latest/contribute/cla/ which is returning a 404.

Maybe the link should just point straight to the [CLA Assistant](https://cla-assistant.io/grafana/grafana) page?

### PR DESCRIPTION
**Pull request by oddlittlebird**
Fixes CLA link

Fixes #23082

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by torkelo** [comment]
need to add the alias in the docs article to fix any external link as well

**Comment by oddlittlebird** [comment]
Added. Please check my syntax. Wasn't sure if I should match `grafana.com/docs/contribute/cla/` or if the `.html` was required. The two that already exist in the file have both.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ When you're ready to contribute, it's time to [Create a pull request](/contribut
 
 #### Contributor License Agreement (CLA)
 
-Before we can accept your pull request, you need to [sign our CLA](https://grafana.com/docs/contribute/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
+Before we can accept your pull request, you need to [sign our CLA](https://cla-assistant.io/grafana/grafana). If you haven't, our CLA assistant prompts you to when you create your pull request.
```
Comment (marefr) [inline CONTRIBUTING.md:61]: Are we planning to remove https://grafana.com/docs/grafana/latest/developers/cla/ page or why do we change this link?

Comment (oddlittlebird) [reply CONTRIBUTING.md:61]: @torkelo pointed out the link as it was was broken, suggested this fix.

Comment (marefr) [reply CONTRIBUTING.md:61]: That link is referenced multiple times in https://grafana.com/docs/grafana/latest/developers/cla/. @torkelo can you please give some input here. PR been open for 21 days and we should get it merged asap.

**Comment by oddlittlebird** [comment]
@aknuds1

**Comment by oddlittlebird** [comment]
For the love of pete, can we please get this dealt with?

**Comment by marefr** [comment]
Reading the issue seems like problem have been resolved if I'm not mistaken?

**Comment by oddlittlebird** [comment]
Guess so.

**Pull request closed without merging**